lua/modules/hyperspace.i

```diff
@@ -112,6 +112,7 @@ namespace std {
 	%template(vector_WeaponMount) vector<WeaponMount>;
 	%template(vector_DamageMessage) vector<DamageMessage*>;
 	%template(vector_Projectile) vector<Projectile*>;
+    %template(vector_MiniProjectile) vector<WeaponBlueprint::MiniProjectile>;
```

Comment:
This whitespace difference bothers me. or at least it looks like a whitespace difference.